source/server/core_service_registrar.cpp

```diff
@@ -1,6 +1,7 @@
 #include "core_service_registrar.h"
```

Comment:
Watch for conflicts with #772

Reply:
I talked with Curtis yesterday, and there are a number of things he's adding in his PR that will help improve the code in my PR, so I'm going to let his go in first.